src/liblog/lib.rs

```diff
@@ -442,19 +440,6 @@ fn init() {
 
         assert!(DIRECTIVES.is_null());
         DIRECTIVES = Box::into_raw(box directives);
-
-        // Schedule the cleanup for the globals for when the runtime exits.
```

Comment:
If this block is to be removed, should we also delete https://github.com/alexcrichton/rust/blob/less-rt/src/liblog/lib.rs#L293 and https://github.com/alexcrichton/rust/blob/less-rt/src/liblog/lib.rs#L386 ?
AFAICT `1` will never be assigned to them in the new code.